entrypoint.sh

```diff
@@ -36,12 +46,13 @@ if [ "$USE_OCI_REGISTRY" == "TRUE" ] || [ "$USE_OCI_REGISTRY" == "true" ]; then
   REGISTRY=$(echo "${REGISTRY_URL}" | awk -F[/:] '{print $4}') # Get registry host from url
   echo "${REGISTRY_ACCESS_TOKEN}" | helm registry login -u ${REGISTRY_USERNAME} --password-stdin ${REGISTRY} # Authenticate registry
   echo "Packaging chart '$CHART_FOLDER'"
-  PKG_RESPONSE=$(helm package $CHART_FOLDER $UPDATE_DEPENDENCIES) # package chart
+  PKG_RESPONSE=$(helm package $CHART_FOLDER $REGISTRY_APPVERSION $REGISTRY_VERSION $UPDATE_DEPENDENCIES) # package chart
   echo "$PKG_RESPONSE"
   CHART_TAR_GZ=$(basename "$PKG_RESPONSE") # extract tar name from helm package stdout
   echo "Pushing chart $CHART_TAR_GZ to '$REGISTRY_URL'"
   helm push "$CHART_TAR_GZ" "$REGISTRY_URL"
   echo "Successfully pushed chart $CHART_TAR_GZ to '$REGISTRY_URL'"
+  rm -rf ${CHART_FOLDER}/charts/*.tgz ${CHART_FOLDER}/*.tgz *.tgz
```

Comment:
This command can fail if there is a lot of *.tgz files, because bash actually expands the asterisk to every matching file, producing a very long command line.
Consider using something like
>find . -name '*.tgz' -delete